### ISSUE TITLE
fix strings, bug in lexer

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#131]
see count_tokens.ts unit test,

```abap
//    {abap: "WRITE `a``b`",                        tokens: 2},
//    {abap: "WRITE `\"foo``\"bar`",                tokens: 2},
```

**Comment by g-back** [comment]
Looks like this was fixed in dc33b5065f57f63a8a5d6e6904048c0066a251b3?

**Comment by larshp** [comment]
yes, thanks, closing
